### PR TITLE
Add docker configuration for fakeserver and documentation on how to r…

### PR DESCRIPTION
**Pull request by anuraaga**
…un it.

The intention is users can use fakeserver even without this repository while developing their first code integrations. Though if they clone the repository they can easily run it too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,26 @@ from the StellarStation Console. Details for registering call credentials on a g
 found [here](https://grpc.io/docs/guides/auth.html). Note that if the key has been revoked on the
 console, it will not be usable to authenticate with the API.
 
-// TODO(rag): Provide details on how to register call credentials for StellarStation private keys.
+The `grpc-auth` and `google-auth-library-oauth2-http` libraries can be used to easily setup
+authentication of an API client.
+
+```java
```
Comment (hoshir) [inline README.md]: Java

File: examples/fakeserver/README.md [hunk 2]
```diff
@@ -13,4 +13,31 @@ integrating with the service.
 
 ## Running
 
-// TODO(rag): Add instructions for running with docker after pushing an image.
+Users of this repository can just use the `run` task.
```
Comment (hoshir) [inline examples/fakeserver/README.md]: Could you sections "Run the fakeserver with Gradle" and "Run the fakeserver with Docker" to clearly say users only need to choose either of ways.

File: README.md (hunk 1 above)
Comment (anuraaga) [reply README.md]: This is just for Github's codeformatting of the snippet, I think it's standard to use all lowercase for it. It's not displayed.

File: examples/fakeserver/README.md (hunk 2 above)
Comment (anuraaga) [reply examples/fakeserver/README.md]: Good point, added subsections

File: README.md (hunk 1 above)
Comment (hoshir) [reply README.md]: Never mind

**Comment by hoshir** [comment]
This is out of scope of this PR. The name "PrintingClient" can be changed to "CommandingClient" which reflects what users want to do.

**Pull request merged**